### PR TITLE
[KMM] Adding an `operator-upgrade` job.

### PR DESCRIPTION
**Pull request by ybettan**

### CODE REVIEW

**Comment by ybettan** [comment]
/pj-rehearse ack
The job is failing because the test file doesn't exist in KMM yet: https://github.com/rh-ecosystem-edge/kernel-module-management/pull/1022

**Comment by ybettan** [comment]
/hold
Will set it to optional until we know it is stable

**Comment by ybettan** [comment]
> /hold
> Will set it to optional until we know it is stable

Done.
/unhold

**Comment by ybettan** [comment]
/hold
This will pull the latest bundle build on Brew which can be for an older z-stream.

**Comment by ybettan** [comment]
/hold
Will verify that that the container has `grpcurl` in it.

**Comment by qbarrand** [comment]
/lgtm
@ybettan please feel free to unhold when you are happy.

File: ci-operator/jobs/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master-postsubmits.yaml [hunk 1]
```diff
@@ -4,12 +4,12 @@ postsubmits:
     always_run: true
     branches:
     - ^master$
-    cluster: multi01
+    cluster: build10
```
Comment (qbarrand) [inline ci-operator/jobs/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master-postsubmits.yaml:7]: What is this?

Comment (ybettan) [reply ci-operator/jobs/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master-postsubmits.yaml]: I have no idea, it was generated automatically by `make ci-operator-config jobs`.

Anyway, I had conflicts on the PR, after rebasing, and re-generating the job file, it disappeared.

**Comment by ybettan** [comment]
/pj-rehearse ack
(test file doesn't exist yet in KMM's repo)
/unhold
/cc @qbarrand 
Ready for merge

**Pull request merged**